setup.py

```diff
@@ -122,23 +122,23 @@ def finalize_options(self):
         pass
 
     def get_npm_name(self):
-        npmName = 'npm';
+        npmName = 'npm'
         if platform.system() == 'Windows':
-            npmName = 'npm.cmd';
+            npmName = 'npm.cmd'
 
-        return npmName;
+        return npmName
 
     def has_npm(self):
-        npmName = self.get_npm_name();
+        npmName = self.get_npm_name()
         try:
             check_call([npmName, '--version'])
             return True
-        except:
+        except BaseException:
             return False
 
     def should_run_npm_install(self):
-        package_json = os.path.join(node_root, 'package.json')
-        node_modules_exists = os.path.exists(self.node_modules)
+        # package_json = os.path.join(node_root, 'package.json')
```

Comment:
@djhoese Did you had something special in mind with this? If not, we could strip the whole function and use `self.has_npm()` in the run-function below.

Reply:
These lines were copied from the jupyter widget cookiecutter template. It is a bug in their template that I think has since been fixed. I think we need to update our usage anyway to be compatible with JLab 3.0+. I went over this change with @Aaru143 and I think it makes sense since they aren't being used.